Documentation/Blazorise.Docs/Pages/Blog/2024-07-19_HowToStyleBlazoriseComponents/Index.md

```diff
@@ -0,0 +1,162 @@
+﻿---
+title: How to style Blazorise components
+description: Discover
+permalink: /blog/how-to-style-blazorise-components
+canonical: /blog/how-to-style-blazorise-components
+image-url: /img/blog/2024-07-19/how-to-style-blazorise-components.png
+image-title: How to style Blazorise components
+author-name: Giorgi
+author-image: giorgi
+posted-on: Jul 19th, 2024
+read-time: 7 min
+---
+
+# Styling Blazorise components
+
+Hello! today we are going to go over how to style Blazorise components!
+
+As your know, Blazorise is a framework-agnostic library, this means we have a lot of options when it comes to choosing what type of framework we want to use.
+
+Check out the [quick start](https://blazorise.com/docs/start) guide here to get started with Blazorise.
+
+## In this article, we will go over the following:
+
+- Styling Blazorise components#How does CSS work with Blazorise?|How does CSS work with Blazorise?
+- Styling Blazorise components#How to style Blazorise components?|How to style Blazorise components?
+- Styling Blazorise components#Limitations of CSS Isolation|Limitations of CSS Isolation
+
+---
+
+So let's dive into the topics and explore our options when it comes to styling Blazorise components.
+
+## How does CSS work with Blazorise?
+
+Blazorise, just like plain Blazor, is a framework that helps us generate responsive web UI. This means that Blazorise supports every CSS property that is supported by the browser. There are no special CSS properties that only apply to Blazor or Blazorise.
```

Comment:
I'm not sure this is an important paragraph? 
But I don't know how to word it differently myself.

Reply:
I could remove it, or rephrase it a bit?

Reply:
Honestly, I'm not sure myself. Use your best judgement.